tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/tests/InstanceMethods.java

```diff
@@ -37,6 +39,9 @@ public class InstanceMethods {
 
     private Integer counter = 0;
 
+    public InstanceMethods() {
```

Comment:
Shall we check whether we need to explicitly have a constructor here? It can use the default constructor.